<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>EyeTracker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/eye.png</normaloff>:/icons/icons/eye.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="playbackGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Playback</string>
    </property>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>210</width>
       <height>240</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="replayButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Replay</string>
     </property>
     <property name="statusTip">
      <string>Replays the selected session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replay</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/replay.png</normaloff>:/icons/icons/replay.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="heatMapButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Heat map</string>
     </property>
     <property name="statusTip">
      <string>Displays heat map for the selected session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Heat map</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/heatmap.png</normaloff>:/icons/icons/heatmap.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="recordGroup">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Record</string>
    </property>
    <widget class="QLabel" name="videoCanvas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(128, 128, 128)</string>
     </property>
     <property name="text">
      <string>Press Start</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Start (Ctrl-S)</string>
     </property>
     <property name="statusTip">
      <string>Starts/stops the video feed from camera...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+S</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="calibrateButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Calibrate (Ctrl-C)</string>
     </property>
     <property name="statusTip">
      <string>Starts calibration process...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/calibrate.png</normaloff>:/icons/icons/calibrate.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+C</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="recordButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>270</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Record (Ctrl-R)</string>
     </property>
     <property name="statusTip">
      <string>Record a new session...</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/icons/record.png</normaloff>:/icons/icons/record.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="shortcut">
      <string>Ctrl+R</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/exit.png</normaloff>:/icons/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/about.png</normaloff>:/icons/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
